autograder/api/config.py

```diff
@@ -299,6 +299,19 @@ def __call__(self, parser, namespace, raw_values, option_string = None):
     required = False,
     parser_options = {'action': 'store_true', 'default': False})
 
+PARAM_PROXY_EMAIL = APIParam('proxy-email',
+    'The email of the user that is the target of this proxy request.',
+    required = True)
+
+PARAM_PROXY_TIME = APIParam('proxy-time',
+    ('The proxy timestamp that will be applied to the request.'
+    + ' By default, a time will be automatically assigned'
+    + ' that does not mark the assignment late.'
```

Comment:
I would just state the options here, like
"the earlier of now and X minutes before the due date".